### PR TITLE
Move to issue forms

### PR DESCRIPTION
**Pull request by Andre601**
## Checks

I have...

- [x] read and understood the [Contributing Guidelines][contributing]
- [x] Updated any nessecary files such as the README.md and/or CHANGELOG.md.

## Type of Pull request

- [ ] **Minor Change**  
  This Pull request doesn't break existing configuration.
- [ ] **Major Change**  
  This Pull request will break existing configuration.
- [ ] **Bug fix**  
  This Pull request will fix a (critical) bug.
- [ ] **Documentation**  
  This Pull request only changes documentation (README.md, CHANGELOG.md, etc.)
- [x] **Other:** `Issue templates` 

## Description

Moved from issue templates to the much better issue forms!
This allows much better designs for issues and also prevents possible problems such as empty issues.

Examples of how they can look like can be found [here](https://github.com/Readme-Workflows/recent-activity/blob/2095ed6d680d8f53e804b494949781f70987223e/.github/ISSUE_TEMPLATE/bug_report.yml) and [here](https://github.com/Readme-Workflows/recent-activity/blob/2095ed6d680d8f53e804b494949781f70987223e/.github/ISSUE_TEMPLATE/feature_request.yml).

Closes #

[contributing]: https://github.com/Readme-Workflows/.github/blob/main/.github/CONTRIBUTING.md

### CODE REVIEW

**Comment by PuneetGopinath** [comment]
@Andre601 Have you tested them?
Is it their in the github docs (please provide link if yes)

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+- type: input
+  attributes:
+    label: Device version
+    description: The current version of your device.
+    placeholder: Windows 10 Pro
+  validations:
+    required: true
+- type: dropdown
+  attributes:
+    label: Browser/App
```
Comment (PuneetGopinath) [inline .github/ISSUE_TEMPLATE/bug_report.yml:92]: Is the Browser/App required?
GitHub actions is gonna run on GitHub's end. So say them to try provide a link to their profile readme's failed workflow

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    - Max
+    - Linux
+    - Android
+    - iOS
+    - Other
+  validations:
+    required: true
+- type: input
+  attributes:
+    label: Device version
```
Comment (PuneetGopinath) [inline .github/ISSUE_TEMPLATE/bug_report.yml:85]: Ditto, as I said below.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    label: Actual behaviour
+    description: What does Recent Activity do instead?
+    placeholder: Recent Activity does...
+  validations:
+    required: true
+- type: dropdown
+  attributes:
+    label: Device
+    description: |-
+      What device did you use?
```
Comment (PuneetGopinath) [inline .github/ISSUE_TEMPLATE/bug_report.yml:71]: Ditto, as below

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    label: Expected behaviour
+    description: What did you expect Recent Activity to do?
+    placeholder: Recent Activity should...
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Actual behaviour
+    description: What does Recent Activity do instead?
+    placeholder: Recent Activity does...
```
Comment (PuneetGopinath) [inline .github/ISSUE_TEMPLATE/bug_report.yml:64]:
```suggestion
    placeholder: Recent Activity does throw an error on ... step
```

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (Andre601) [reply .github/ISSUE_TEMPLATE/bug_report.yml:92]: This is a one to one copy from the MD file, where you had sections about OS and browser...

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 2 above)
Comment (Andre601) [reply .github/ISSUE_TEMPLATE/bug_report.yml:85]: *as said above

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 3 above)
Comment (Andre601) [reply .github/ISSUE_TEMPLATE/bug_report.yml:71]: *as said above

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 4 above)
Comment (Andre601) [reply .github/ISSUE_TEMPLATE/bug_report.yml:64]: Is that really needed? A bug report is not always about errors. Not committing this...

Comment (PuneetGopinath) [reply .github/ISSUE_TEMPLATE/bug_report.yml:64]: Okay

**Review by PuneetGopinath — APPROVED** [review]

**Comment by PuneetGopinath** [comment]
> @Andre601 Have you tested them?
> Is it their in the github docs (please provide link if yes)

What do you say for this?

**Comment by Andre601** [comment]
> > @Andre601 Have you tested them?
> > Is it their in the github docs (please provide link if yes)
> 
> What do you say for this?

They should work...
Like I use issue forms on several places without issue and GitHub does perform some validation checks and would warn you about malformed YAML values.

**Review by PuneetGopinath — APPROVED** [review]

**Comment by Andre601** [comment]
Here are the docs btw:
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

**Comment by abhijoshi2k** [comment]
Is this PR Ready?

**Pull request merged**